alphabet/alphabet_contract.go

```diff
@@ -71,8 +71,11 @@ func _deploy(data any, isUpdate bool) {
 		panic("non-notary mode is not supported anymore")
 	}
 
-	if len(args.addrNetmap) != interop.Hash160Len || len(args.addrProxy) != interop.Hash160Len {
-		panic("incorrect length of contract script hash")
+	if len(args.addrNetmap) != interop.Hash160Len {
+		args.addrNetmap = common.ResolveFSContract("netmap")
```

Comment:
The behavior now is to derive address from NNS, so in most cases it will just get a proper hash. In the future non-NNS logic can be omitted completely. I'd leave it as is for now, tools that care pass correct things and we resolve from NNS otherwise.